test/browser/spec/ol/source/raster.test.js

```diff
@@ -47,7 +47,7 @@ where('Uint8ClampedArray').describe('ol.source.Raster', function () {
     greenSource = new Static({
       url: green,
       imageExtent: extent,
-      attributions: ['green raster source'],
+      attributions: (frameState) => 'green raster source',
```

Comment:
Since the array of string flavour should still work, it would be better to add another source to this test case or create a separate test case for the function flavour.

Reply:
`redSource` above still uses an array of strings, and they are used in combination, so I was hoping it'd be ok.

Reply:
Oh, sorry! I just looked at the diff, not the context. In this case it's fine.